.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Deploy Docs
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-20.04
+    env:
+      MIKE_VERSION: 0.9
+      MIKE_ALIAS: latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install npm dependencies
+      run: npm ci
+
+    - name: Update CLI Doc
+      run: npm run generate-cli-doc
+
+    - name: GitHub Configuration
+      run: |
+        git fetch origin gh-pages --depth=1
+        git config --local user.email "github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
+
+    - name: Build Mike Image
+      run: |
+        chmod +x ./scripts/buildImage.sh
+        ./scripts/buildImage.sh
+
+    - name: Build MkDocs with Mike (versioning)
+      run: docker run --rm -v $(pwd):/docs squidfunk/mkdocs-material:8.5.9 mike deploy $MIKE_VERSION $MIKE_ALIAS --rebase --update-aliases
```

Comment:
Specifying the mike version in a env variable https://docs.github.com/en/actions/learn-github-actions/environment-variables. So the version has to be maintained only once

Reply:
I'm not quite sure I understand this one. Could you elaborate a bit more?

I think it's already been done and reused here and below. Please take a look at line #10:

>     env:
>       MIKE_VERSION: 0.9
>       MIKE_ALIAS: latest

Is this the env variable you refer to?

Reply:
Sorry I meant the version of the `squidfunk/mkdocs-material:8.5.9`. This could be stored in a nenv

Reply:
Done